src/Stratis.Bitcoin/FullNode.cs

```diff
@@ -277,14 +277,15 @@ public void Dispose()
 
             this.ConnectionManager.Dispose();
             this.bestChainSelector.Dispose();
-            this.loggerFactory.Dispose();
 
             foreach (IDisposable disposable in this.Resources)
                 disposable.Dispose();
 
             // Fire the NodeFeatureExecutor.Stop.
             this.fullNodeFeatureExecutor.Dispose();
 
+            this.loggerFactory.Dispose();
```

Comment:
Hmm this is odd, loggerFactory needs to be disposed last so I find it weird it was not already here(I remember putting it here some time ago)

In any case there is only one logger factory so not need to dispose of it several times, so remove it from the `this.Resources`

Reply:
This should be replaced with disposal of node settings

Reply:
I don't think it is added to this.Resources, and yes agreed I should dispose of the nodeSettings instead